script.js

```diff
@@ -22,49 +24,59 @@ water.addEventListener('click', function() {
   // apple[randNum].setAttribute('src', 'images/apple-rotten.png')
   // You also already have the hieght as
   // apple[0].style.height, so instead fo storing it as a var you could just check that value
-  if (randomNumber === 0) {
-    if (apple1height === 18) {
-      apple[0].style.height = '18vh'
-      apple[0].setAttribute('src', 'images/apple-rotten.png')
-    } else {
-      apple1height += 3
-      apple[0].style.height = apple1height + 'vh'
-    }
-  } else if (randomNumber === 1) {
-    if (apple2height === 18) {
-      apple[1].style.height = '18vh'
-      apple[1].setAttribute('src', 'images/apple-rotten.png')
-    } else {
-      apple2height += 3
-      apple[1].style.height = apple2height + 'vh'
-    }
-  } else if (randomNumber === 2) {
-    if (apple3height === 18) {
-      apple[2].style.height = '18vh'
-      apple[2].setAttribute('src', 'images/apple-rotten.png')
-    } else {
-      apple3height += 3
-      apple[2].style.height = apple3height + 'vh'
-    }
-  } else if (randomNumber === 3) {
-    if (apple4height === 18) {
-      apple[3].style.height = '18vh'
-      apple[3].setAttribute('src', 'images/apple-rotten.png')
-    } else {
-      apple4height += 3
-      apple[3].style.height = apple4height + 'vh'
-    }
-  } else if (randomNumber === 4) {
-    if (apple5height === 18) {
-      apple[4].style.height = '18vh'
-      apple[4].setAttribute('src', 'images/apple-rotten.png')
+
+  function growApples(randomNumber) {
+    if (apple[randomNumber].style.height === '18vh') {
+      apple[randomNumber].setAttribute('src', 'images/apple-rotten.png')
     } else {
-      apple5height += 3
-      apple[4].style.height = apple5height + 'vh'
+      apple[randomNumber].style.height = parseInt(apple[randomNumber].style.height) + 3 + 'vh'
```

Comment:
On the first attempt `apple[randomNumber].style.height` comes up as `undefined` or `''`. Therefore you can't parseInt it or plus things on to it.

You need to have a check to see if `apple[randomNumber].style.height` is defined, and if its not do something slightly differently.

Otherwise you could define `apple[randomNumber].style.height` by having `style="height: 0vh"` in every apple in the HTML.